iib/workers/tasks/utils.py

```diff
@@ -447,7 +412,13 @@ def set_registry_auths(registry_auths):
         reset_docker_config()
 
 
-@retry(wait_on=IIBError, logger=log)
+@retry(
+    exceptions=IIBError,
+    tries=get_worker_config().iib_total_attempts,
+    delay=5,
```

Comment:
Could you explain why you chose these numbers?

Reply:
Because it looked reasonable to me. The delay will be 5, 10, 20, 40 and 80 seconds. 
Unfortunately I do not have any information about the time - how long does it take to recover.